test/command/suite/normalizers/nfkc/unify_alphabet_diacritical_mark.expected

```diff
@@ -1,13 +1,74 @@
-normalize   'NormalizerNFKC("unify_alphabet_diacritical_mark", true)'   "ÀÁÂ"   WITH_TYPES
+normalize   'NormalizerNFKC("unify_alphabet_diacritical_mark", true)'   "ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖ×ØÙÚÛÜÝÞßàáâãäåæçèéêëìíîïðñòóôõö÷øùúûüýþÿ"   WITH_TYPES
 [
   [
     0,
     0.0,
     0.0
   ],
   {
-    "normalized": "àáâ",
+    "normalized": "aaaaaaæceeeeiiiiðnooooo×øuuuuyþßaaaaaaæceeeeiiiiðnooooo÷øuuuuyþy",
```

Comment:
It's difficult to check whether the normalized text is correct or not.
Could you split this test to `test/command/suite/normalizers/nfkc/unify_alphabet_diacritical_mark/[a-z].{test,expected}`?

Reply:
Thanks 🙏🏾 @komainu8 told me the same thing too.
Sure, I will split test cases into some files.